### ISSUE TITLE
No such file or directory: '' error during the training step

### ISSUE THREAD

**Issue by sebastien-clara** [mozi1924/Qwen3-TTS-EasyFinetuning#1]
Windows 11
DockerDesktop 4.60.1
Using the CLI

I'm getting a "No such file or directory:" error during the training step.
I've attached the commands I'm running and the output they return in the terminal.

> docker run --rm --gpus all -v C:\Users\bot\Downloads\playlist\Qwen3-TTS-EasyFinetuning-data:/workspace ghcr.io/mozi1924/qwen3-tts-easyfinetuning python /workspace/src/cli.py prepare --input_dir /workspace/raw-dataset --speaker_name Ella --experiment_name expElla

==========
== CUDA ==
==========

CUDA Version 12.4.1

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

════════════════════════════════════════════════════════════
  🎙️ Qwen3-TTS Full Data Preparation
════════════════════════════════════════════════════════════

  ▶ Step 1: Audio Split & Resample
    [██████████████████████████░░░░]  89.1% Resampling 44/44gments using 19 cores...
    ✅ Successfully split and resampled 44 segments.

  ▶ Step 2: ASR Transcription & Cleaning
Found local model at /workspace/models/Qwen/Qwen3-ASR-1.7B, skipping download!
    [░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   1.0% Loading ASR Model: /workspace/models/Qwen/Qwen3-ASR-1.7B...We will use 90% of the memory on device 0 for storing the model, and 10% for the buffer to avoid OOM. You can set `max_memory` in to a higher value to use more memory (at your own risk).
Loading checkpoint shards: 100%|██████████| 2/2 [00:27<00:00, 13.94s/it]
The following generation flags are not valid and may be ignored: ['temperature']. Set `TRANSFORMERS_VERBOSITY=info` for more details.
    [███░░░░░░░░░░░░░░░░░░░░░░░░░░░]  10.0% Transcribing batch 1/3...Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
    [███████████░░░░░░░░░░░░░░░░░░░]  38.3% Transcribing batch 2/3...Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
    [████████████████████░░░░░░░░░░]  66.7% Transcribing batch 3/3...Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
    [█████████████████████████████░]  98.0% Writing JSONL data...
    ✅ Successfully transcribed 44 segments to /workspace/final-dataset/Ella/tts_train.jsonl.

  ▶ Step 3: Data Tokenization
Found local model at /workspace/models/Qwen/Qwen3-TTS-Tokenizer-12Hz, skipping download!
    [█░░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   5.0% Loading Tokenizer: /workspace/models/Qwen/Qwen3-TTS-Tokenizer-12Hz...encoder_config is None. Initializing encoder with default values
decoder_config is None. Initializing decoder with default values
    [████████████████████████████░░]  95.0% Saving JSONL output...of 44 files...
    ✅ Successfully tokenized 44 entries.

════════════════════════════════════════════════════════════
  ✅ Full Preparation Complete!
════════════════════════════════════════════════════════════

> docker run --rm --gpus all -v C:\Users\bot\Downloads\playlist\Qwen3-TTS-EasyFinetuning-data:/workspace ghcr.io/mozi1924/qwen3-tts-easyfinetuning python /workspace/src/cli.py train --experiment_name expElla --speaker_name Ella --epochs 3

==========
== CUDA ==
==========

CUDA Version 12.4.1

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

`torch_dtype` is deprecated! Use `dtype` instead!
You are attempting to use Flash Attention 2 without specifying a torch dtype. This might lead to unexpected behaviour
/workspace/src/dataset.py:46: UserWarning: PySoundFile failed. Trying audioread instead.
  audio, sr = librosa.load(x, sr=24000, mono=True)
/opt/conda/lib/python3.11/site-packages/librosa/core/audio.py:184: FutureWarning: librosa.core.audio.__audioread_load
        Deprecated as of librosa version 0.10.0.
        It will be removed in librosa version 1.0.
  y, sr_native = __audioread_load(path, offset, duration, dtype)

════════════════════════════════════════════════════════════
  🏋️ Qwen3-TTS Fine-tuning
════════════════════════════════════════════════════════════
  Experiment   : expElla
  Speaker      : Ella
  Base Model   : Qwen/Qwen3-TTS-12Hz-0.6B-Base
  GPU Device   : cuda:0
  Batch Size   : 2
  Learning Rate: 1e-07
  Epochs       : 3
  Grad Accum   : 4
Found local model at /workspace/models/Qwen/Qwen3-TTS-12Hz-0.6B-Base, skipping download!

  ▶ Starting TensorBoard on port 6006...

  ▶ Training started on cuda:0...
Multi-speaker mode: 1 speakers
  Speaker 'Ella' -> spk_id 3000
Dataset size: 44
Dataloader length: 22
Global step starts at: 0

    📈 Epoch 0 | Starting Training...

    ❌ Unhandled exception in training: [Errno 2] No such file or directory: ''

════════════════════════════════════════════════════════════
  ✅ Training Complete!
════════════════════════════════════════════════════════════
  Output: /workspace/output/expElla
